Combobox.js

```diff
@@ -482,8 +480,9 @@ define([
 				
 				if (!initValueSingleMode()) {
 					// List not ready, wait.
-					this.list.on("query-success", function () {
+					var waitListener = this.list.on("query-success", function () {
```

Comment:
The variable waitListener is needed only to be able to remove the listener after the first initialisation.
It was not needed before because the firstItemRenderer was already loaded when the function was called. Also the ```initValueSingleMode()``` function was always returning true and the listener was not defined.
The problem was when modifying the query (with autoFilter === true), when supressing a character the function ```initValueListener()``` was automatically called and the label of the first item was refreshed. Then we could not see the query anymore.